src/Button/MeasureButton/MeasureButton.tsx

```diff
@@ -289,15 +295,15 @@ class MeasureButton extends React.Component<MeasureButtonProps> {
       this._drawInteraction.setActive(pressed);
 
       this._eventKeys.drawstart = this._drawInteraction.on(
-        'drawstart', this.onDrawStart, this
+        'drawstart', e => this.onDrawStart(e)
```

Comment:
Can you explain, why you changed this?

Reply:
in openlayers v5 the opt_this parameter on `on` was removed